api_v3/services/MediaService.php

```diff
@@ -149,6 +149,16 @@ protected function replaceResource(KalturaResource $resource, entry $dbEntry, $c
 		}
 		else
 		{
+			$kResource = $resource->toObject();
+			if ( $kResource instanceof kOperationResource ) {
+				$internalResource = $kResource->getResource();
+				if ( $dbEntry->getIsTrimDisabled()
```

Comment:
Let's talk about this. I didn't write this code and would be happy to change whatever needs to be.